Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,15 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Pathname(".").find do |f|
+        next unless f.file?
+        next unless File.foreach(f).first(1).include?("/usr/bin/env python")
```

Comment:
I'd prefer if this used the same approach as I suggested and used the same regex to match as to do the replace. As-is this is not reliable/robust.

Reply:
I just changed it to only read the first line for performance reasons. I did not change the regex, I'll come back to this, I still need to read all your comments :)

Reply:
My version was more efficient as it did not read the whole first line but instead just the minimum number of characters necessary to match the regex 😉

Reply:
Note: reading the first line in (e.g. binary files) without an EOL character will involve reading the entire file.